xbmc/addons/interfaces/Addon/AddonCallbacksAddon.cpp

```diff
@@ -77,6 +77,7 @@ CAddonCallbacksAddon::CAddonCallbacksAddon(CAddon* addon)
   m_callbacks->FileExists         = FileExists;
   m_callbacks->StatFile           = StatFile;
   m_callbacks->GetFileProperty    = GetFileProperty;
```

Comment:
`GetFilePropertyValue` (see my other comment re the naming)

Reply:
GetFilePropertyValues (!!!)

Reply:
No. We have two API functions now. One for single name - single value => GetFilePropertyValue
One for single name - multi values => GetFilePropertyValues

Reply:
No. We have two API functions now. One for single name - single value, one for single name - multi values. Please look at the full diff and you will see what I mean.